composer.json

```diff
@@ -2,8 +2,18 @@
     "name": "yola/yophpcs",
     "description": "Yola.com PHP Code Standard",
     "minimum-stability": "dev",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/devshep/codeception.git"
```

Comment:
Probably best thing to do is to create a ticket to switch back the dependency when Codeception does a version bump. They haven't done it yet, but I don't think it's worth waiting on them before we move forward.